app/visit/components/checkIn.js

```diff
@@ -22,6 +22,7 @@
 
     vm.$onChanges = $onChanges;
     vm.checkIn = checkIn;
+    vm.cancelCheckIn = cancelCheckIn;
 
     function $onChanges(changesObj) {
       var patient = changesObj.patient.currentValue;
```

Comment:
Is vm.disableCheckin still used? Line 34 sets it but it looks like all other references were deleted in this check in.

Reply:
Good catch, it's not used anymore. I had removed it [here](https://github.com/esaude/esaude-emr-poc/pull/540/commits/e99eb8d50899b7d4a4dfbe52dcfd402dc6ab54e6) but I guess it came back when I merged `master` into this branch.